src/adapters/store/postgres.ts

```diff
@@ -94,6 +94,13 @@ export function createPgStoreAdapter(pool: Pool, logger: Logger = console): Stor
     ).then((results) => Option.of(results.rows[0]));
   }
 
+  async function exists(id: string) {
```

Comment:
Please add a return type here. It's a small function so that might be obvious, but it's better to be explicit.

Reply:
Return type added.